src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPCompletionFeature.java

```diff
@@ -97,7 +97,7 @@ public boolean isCompletionSupported(@NotNull PsiFile file) {
      * @param file the file.
      * @return true the file associated with a language server can support resolve completion and false otherwise.
      */
-    public boolean isResolveCompletionSupported(@Nullable PsiFile file) {
+    public boolean isResolveCompletionSupported(@NotNull PsiFile file) {

```

Comment:
All callers were passing non-null values and the `isResolveCompletionSupported()` expects a non-null argument, so I changed the nullability annotation to match reality.